--- a/sysdeps/unix/sysv/linux/sys/mount.h	2026-04-23 15:00:41.454757534 -0700
+++ b/sysdeps/unix/sysv/linux/sys/mount.h	2026-04-23 15:01:11.984229598 -0700
@@ -265,9 +265,10 @@
 #define FSOPEN_CLOEXEC          0x00000001
 
 /* open_tree flags.  */
+#ifndef OPEN_TREE_CLONE
 #define OPEN_TREE_CLONE    1         /* Clone the target tree and attach the clone */
 #define OPEN_TREE_CLOEXEC  O_CLOEXEC /* Close the file on execve() */
-
+#endif
 
 __BEGIN_DECLS
 
